### PR TITLE
Update domain names

### PR DESCRIPTION
**Pull request by MJafarMashhadi**
Added `alchemy.impact.georgian.io`

### CODE REVIEW

File: docker-compose-staging.yml [hunk 1]
```diff
@@ -74,7 +74,7 @@ services:
     env_file:
       - .env
     environment:
-      - DOMAIN_NAME=alchemy.impact.georgianpartners.com
+      - DOMAIN_NAME=alchemy.impact.georgianpartners.com alchemy.impact.georgian.io
```
Comment (jzhang-gp) [inline docker-compose-staging.yml:77]: Now I think about it, can we move the domain name to a environment variable that's not public? This is an internal domain which is not applicable to everyone else.

Comment (MJafarMashhadi) [reply docker-compose-staging.yml:77]: You mean moving it to the `.env` file?

Comment (jzhang-gp) [reply docker-compose-staging.yml:77]: If that works without exposing the url in the public, yes.

Comment (MJafarMashhadi) [reply docker-compose-staging.yml:77]: I should check to see if it works, nginx container uses the env vars to preprocess the config template to a config file.

Comment (MJafarMashhadi) [reply docker-compose-staging.yml:77]: If it worked I'll update this PR to remove this line from compose file

Comment (MJafarMashhadi) [reply docker-compose-staging.yml]: Worked ^_^ @jzhang-gp

**Review by jzhang-gp — APPROVED** [review]
LGTM

**Pull request merged**